### DISCUSSION TITLE
Is it possible to create own module ?

### DISCUSSION THREAD

**Discussion by shankhadevpadam** [lunarphp/lunar#1257] [Q&A]
I am building my e-commerce using Lunar. So how can I make CMS for my website like a blog?

**Comment by glennjacobs** [comment]
You can certainly make your own functionality, or you can use something like Statamic.

**Reply by shankhadevpadam** [reply]
I mean how to append a blog module in Lunar? Is it possible?

**Reply by glennjacobs** [reply]
Maybe you want this? https://docs.lunarphp.io/admin-hub/extending.html#adding-to-menus

**Reply by shankhadevpadam** [reply]
Thanks and one final question when will the filament version be released?

**Reply by glennjacobs** [reply]
We don't have a timeline for that currently. As soon as we have more information, it will be published.
